CMakeLists.txt

```diff
@@ -17,6 +17,12 @@ add_custom_target(add-copyright
   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   COMMENT "Add copyright")
 
+add_custom_target(build-version
```

Comment:
Just a reminder that you will have to make any target that uses build-version depend on this target.